L78-python-rich-server-context.md

```diff
@@ -0,0 +1,73 @@
+Title
+----
+* Author(s): Chris St. Pierre
```

Comment:
Please use your GitHub handle. You can add your real full name in parentheses if you'd like.